src/rust/inetstack/protocols/layer4/tcp/tests/simulator.rs

```diff
@@ -492,9 +492,23 @@ impl Simulation {
         };
 
         let syscall_qd: QDesc = match args.qd {
-            Some(qd) => qd.into(),
-            None => {
-                anyhow::bail!("remote queue descriptor must have been previously assigned");
+            // 500 in the annotation will represent the local queue descriptor number because they are allocated
+            // starting at 500.
+            Some(qd) if qd == 500 => match self.local_qd {
```

Comment:
What do you think about storing the value in a variable? It would be safe against more usage in test going fwd + it's already used multiple times in this file.

Reply:
It is stored in a variable, self.local_qd. It's just that the trace file doesn't have variables and in order to add them, I'd have to make major changes to the parser, so I figured I wouldn't do that in this PR.

Reply:
Ok, sounds good. We can skip this.